sql/src/main/java/io/crate/analyze/WindowDefinition.java

```diff
@@ -58,14 +59,20 @@ public WindowDefinition(List<Symbol> partitions, OrderBy orderBy, WindowFrameDef
         this.windowFrameDefinition = windowFrameDefinition;
     }
 
+    public static WindowDefinition emptyWindow() {
```

Comment:
Can we move this into the test sources? In the proper "production" case, it doesn't make sense to create empty window functions I think?